Move io tasks

mlubej opened the pull request:
There is one more task which I have not ported, the `SentinelHubSen2corTask`. I think it pulls in too much code for a very simple thing, which is now easier achievable with the new code, so IMHO the `SentinelHubSen2corTask` has become redundant and I would just deprecated/remove it in eo-learn.

zigaLuksic reviewed (commented):
Could you also include the `GeoDBVectorImportTask` here? it's in io.extras, which is why we probably missed it

mlubej commented:
> Could you also include the GeoDBVectorImportTask here? it's in io.extras, which is why we probably missed it

added.

also, @batic, I moved the Meteoblue related tasks, but I only found them used in the https://github.com/sentinel-hub/eo-learn-examples/blob/08f8bb349efc47595b7d7525403c36265ed5cfe4/GEM-data/weather-data.ipynb notebook, so it was a simple change.
